Makefile

```diff
@@ -179,7 +183,7 @@ client: ${PREFIX}/bin/notary
 binaries: ${PREFIX}/bin/notary-server ${PREFIX}/bin/notary ${PREFIX}/bin/notary-signer
 	@echo "+ $@"
 
-static: ${PREFIX}/bin/static/notary-server ${PREFIX}/bin/static/notary-signer
+static: ${PREFIX}/bin/static/notary-server ${PREFIX}/bin/static/notary-signer ${PREFIX}/bin/static/notary
```

Comment:
Wondering if we should also add a target for `${PREFIX}/bin/static/notary` in the darwin section of the `ifeq`?  Otherwise on a Mac I get:

```
make static
pkcs11 import was not found anywhere without a build tag, yay
notary-server: static builds not supported on OS X
notary-signer: static builds not supported on OS X
make: *** No rule to make target `/Users/cyli/Go/src/github.com/docker/notary/bin/static/notary', needed by `static'.  Stop.
```

Might just be nice to have a message instead of the error - or perhaps a better error.  :)